README.md

```diff
@@ -3,13 +3,97 @@
 
-## How does it work?
+## Usage
+
+The main Babel packages only transform JavaScript _syntax_: you also need to load a polyfill, to make native _functions_ (`Array.prototype.flat`) or _built-in objects_ (`Promise`) work in older browsers. The Babel plugins implemented in this repository automatically inject these polyfills in your code, while trying to only load what is really needed. It does this based on your compilation targets and on what you are using in your code.
+
+These plugins (we are calling them "polyfill providers") support different [injection *methods*](#injection-methods), to better fit your needs.
+
+For example, if you want to inject imports to [`es-shims`](https://github.com/es-shims) polyfills by adding the missing functions to the global objects, you could configure Babel as such:
+
+<!-- prettier-ignore-start -->
+<table>
+<thead><tr>
+<th align="center">Configuration</th>
+<th align="center">Input code</th>
+<th align="center">Output code</th>
+</tr></thead>
+<tr>
+<td>
+
+```json
+{
+  "plugins": [
+    ["polyfill-es-shims", {
+      "method": "usage-global",
+      "targets": {
+        "firefox": 65
+      }
+    }]
+  ]
+}
+```
+
+</td>
+<td>
+
+```js
+ 
+ 
+ 
+Promise.allSettled([
+  p1,
+  p2
+]).finally(() => {
+  console.log("Done!");
+});
+ 
+```
+
+</td>
+<td>
+
+```js
+import "promise.prototype.finally/auto.js";
+import "promise.allsettled/auto.js";
+
+Promise.allSettled([
+  p1,
+  p2
+]).finally(() => {
+  console.log("Done!");
+});
+ 
+```
+
+</td>
+</tr>
+</table>
+<!-- prettier-ignore-end -->
+
+If you want to see more configuration examples, you can check the migration docs: [`docs/migration.md`](https://github.com/babel/babel-polyfills/blob/master/docs/migration.md).
+
+If you are interested in reading about all the options supported by these plugins, you can check the usage docs: [`docs/usage.md`](https://github.com/babel/babel-polyfills/blob/master/docs/migration.md).
+
+## Supported polyfills
+
+<!--prettier-ignore -->
+| Polyfill | Plugin | Methods |
+| :------: | :----: | :-----: |
+| `core-js@2` | [`babel-plugin-polyfill-corejs2`](./packages/babel-plugin-polyfill-corejs2) | `entry-global`, `usage-global` and `usage-pure` |
+| `core-js@3` | [`babel-plugin-polyfill-corejs3`](./packages/babel-plugin-polyfill-corejs3) | `entry-global`, `usage-global` and `usage-pure` |
+| `es-shims` | [`babel-plugin-polyfill-es-shims`](./packages/babel-plugin-polyfill-es-shims) | `usage-global` and `usage-pure` |
+| `regenerator-runtime` | [`babel-plugin-polyfill-regenerator`](./packages/babel-plugin-polyfill-regenerator) | `entry-global`, `usage-global` and `usage-pure` |
+
+> 💡 If you want to implement support for a custom polyfill, you can use `@babel/helper-define-polyfill-provider`. ([`docs/polyfill-provider.md`](https://github.com/babel/babel-polyfills/blob/master/docs/polyfill-provider.md).)
```

Comment:
You basically said this here, but above the table should we say, these are the ones we've made/maintained/setup but doesn't mean you have to use these? Or a note we can list others if it meets some criteria